.github/actions/validate-with-schema/README.md

```diff
@@ -8,7 +8,7 @@ This action validates a `json` or `yaml` file using a given schema. Essentially
 | ---- | ---- | ----------- | -------- | ------- | ------- |
 | `schema-version` | `string` | Version of the schema required in SchemaVer | `true` | N/A | `"1-0-0"` |
 | `schema-location` | `string` | Directory within access-nri/schema that contains the schema | `true` | N/A | `"access-nri.org.au/deployment/config/versions"` |
-| `data-location` | `string` | Path to the json/yaml data in the callers repository | `true` | N/A | `"./some/data.json"` |
+| `data-location` | `string` | Path(s) to the json/yaml data file(s) in the callers repository. File is either `.json` or `.yml`/`.yaml` | `true` | N/A | `"./some/data.json"`, `"/path/to/some/data.yaml"` |
```

Comment:
Just checking, the paths should be comma separated?

In the docs it says

> List of file paths to validate. Each file path must be on a newline.

I don't think you can pass an input with newlines, but could use `tr` to achieve this, but would definitely break if there was a comma in a filename, so that should be explicitly *not* allowed.

Reply:
You can pass an input with newlines, with `|`:
```yaml
uses: some-action@v1
with:
  some-input: |
    some
    multi
    line
    thing
  some-other-input: blah
```

Reply:
So this needs to be explicitly shown in the docs I think. Currently it is a comma separated list.

Reply:
Done!